src/transformers/utils/dummy_timm_and_vision_objects.py

```diff
@@ -57,37 +57,6 @@ def __init__(self, *args, **kwargs):
         requires_backends(self, ["timm", "vision"])
 
 
-DETR_PRETRAINED_MODEL_ARCHIVE_LIST = None
```

Comment:
This file should be entirely removed as there are no more timm and vision dummies necessary.